src/TimestampController.ts

```diff
@@ -56,6 +56,7 @@ class TimestampController {
   private readonly contract: TypedContract<typeof TRUSTED_HINT_REGISTRY_ABI>;
   private merkleTree?: StandardMerkleTree<any[]>;
   private rootHash?: string;
+  private encoding: string[];
```

Comment:
```suggestion
  private encoding: string[] = [];
```

if its not optional, we can assign it to an empty array? or should we some initialize it in the constructor